boards/shields/g1120b0mipi/g1120b0mipi.overlay

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	aliases {
+		kscan0 = &kscan_input;
+	};
+
+	chosen {
+		zephyr,display = &rm67162_g1120b0mipi;
+		zephyr,keyboard-scan = &kscan_input;
+	};
+
+	en_mipi_display_g1120b0mipi: enable-mipi-display {
+		compatible = "regulator-fixed";
+		regulator-name = "en_mipi_display";
+		enable-gpios = <&nxp_mipi_connector 32 GPIO_ACTIVE_HIGH>;
+		regulator-boot-on;
+	};
+};
+
+&nxp_mipi_i2c {
+	status = "okay";
+	ft3267_g1120b0mipi: ft3267@38 {
+		/*
+		 * Note- the actual controller present on this IC is a FT3267,
+		 * but the FT35336 driver in Zephyr supports this IC.
+		 */
+		compatible = "focaltech,ft5336";
+		reg = <0x38>;
+		int-gpios = <&nxp_mipi_connector 29 GPIO_ACTIVE_LOW>;
+		reset-gpios = <&nxp_mipi_connector 28 GPIO_ACTIVE_HIGH>;
+		kscan_input: kscan-input {
```

Comment:
the nodelabel shoudl be something like `kscan_input_ft3267`, that's the format for shields IIRC, I think to avoid colliding with a board one

Reply:
I think @erwango has suggested `<device_name>_<shield_name>` in the past: https://github.com/zephyrproject-rtos/zephyr/pull/60174#discussion_r1260694694, so that's what I've changed to here.